c/pthread-driver-races/char_generic_nvram_nvram_llseek_nvram_unlocked_ioctl_true-unreach-call.i

```diff
@@ -6498,6 +6498,416 @@ extern void nvram_sync(void);
 extern ssize_t nvram_get_size(void);
 extern unsigned char nvram_read_byte(int i);
 extern void nvram_write_byte(unsigned char c, int i);
+
+const unsigned int default_alloc_size = 4;
+
+// Function: external_alloc
+// with type: void *external_alloc(unsigned int size)
+// with return type: void *
+void *external_alloc(unsigned int size)
+{
+	return malloc(size);
```

Comment:
In line with @PhilippWendler's comment:
```suggestion
	char *p = malloc(size);
	for(unsigned i = 0; i < size; ++i)
	  p[i] = __VERIFIER_nondet_char();
	return p;
```

Reply:
> I guess that often the calling code will read from this pointer, which would be undefined behavior.

Of course, some initialization may be better than nothing, but we still can read incorrect values with such workaround initialization. If we need a correctly initialized memory, especially for structures, I suggest to have initialization for each corresponding type.